cards/cardwithoverlay.css

```diff
@@ -0,0 +1,108 @@
+:root{
+    --card-background-color:aliceblue;
+    --card-border-color:black;
+    --card-badge-bg-col:#BFC205;
+    --card-badge-rating-bg:green;
+    --card-primary-color:#F6830F;
+}
+*{
+padding:0;
+margin:0;
+box-sizing: border-box;
+font-family: sans-serif;
+letter-spacing: 1px;
+}
+
+.card-legend{
+width:20%;
+border:1px solid var(--card-border-color);
+background-color: var(--card-background-color);
+position: relative;
+border-radius:10px;
+overflow:hidden;
+
+}
+.card-overlay:hover{
+    background-color: rgba(0,0,0,0.5);
+    z-index:1;
+    cursor:pointer;
+}
+    
+
+.para-card .heading-card{
+padding-top:0.2rem;
+padding-bottom:0.2rem;
+font-weight: 700;
+margin-left:0.3rem;
+}
+.para-card .subheading-card{
+padding-top:0.2rem;
+padding-bottom:0.2rem;
+font-weight: 200;
+margin-left:0.3rem;
+}
+.para-card .price-card{
+position: absolute;
+right:8px;
+bottom:45px;
+font-weight: 200;
+}
+.para-card .rating-card .rating-h{
+background-color: var(--card-badge-rating-bg);
+display: inline;
+padding:0.3rem;
+padding-left:0.5rem;
+padding-right:0.5rem;
```

Comment:
You can use shorthand property like padding: 0.3rem 0.5rem; 
It will give the same effect.